scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -174,3 +174,124 @@ <h5 class="header">General Institute Settings</h5>
     </tbody>
   </table>
 {% endmacro %}
+
+
+{% macro variant_list_content(institute, variants, acmg_map) %}
+<div class="card mt-3">
+  <div class="card-body">
+    <table id="verified_table" class="table display table-sm">
+      <caption>List of selected variants</caption>
+      <thead class="thead table-light sticky-offset">
+        <tr>
+          <th>Gene</th>
+          <th>Variant</th>
+          <th>Change</th>
+          <th>HGVS[c]</th>
+          <th>HGVS[p]</th>
+          <th>Category</th>
+          <th data-bs-toggle='tooltip' data-bs-container='body' title="score(model version)">Rank score</th>
+          <th data-bs-toggle='tooltip' data-bs-container='body' title="ref/alt-GQ">Zygosity</th>
+          <th>Inheritance</th>
+          <th>ACMG</th>
+          <th>Case</th>
+          <th>Validated status</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for variant in variants %}
+          {% set case = variant["case_obj"] %}
+          <tr>
+            <td>
+              <span class="text-body">{{ variant.hgnc_symbols|join(', ') if variant.hgnc_symbols else '--'}}</span>
+            </td>
+            <td>
+              <a href="{{ url_for('variant.variant',
+                  institute_id=institute._id,
+                  case_name=case.display_name,
+                  variant_id=variant._id) }}" data-bs-toggle='tooltip' data-bs-container='body' title="{{ variant.display_name }}">{{ variant.display_name|truncate(15, True) }}
+              </a>
+            </td>
+            <td>
+              <span class="text-muted" style="display: inline-block; margin-left: 10px;" data-bs-toggle='tooltip' data-bs-container='body' title="{{ variant.reference|truncate(30, True) }} → {{ variant.alternative|truncate(30, True) }}">
```

Comment:
There's a mix of quoting styles here, " and '. Might that be part of what is confusing the excel export script?

Reply:
I think it might be the ">" character, check this out: https://datatables.net/forums/discussion/30014/problem-with-jquery-datatables-buttons-excel-export-using-special-character-like-or

I've spent many hours trying the suggested solutions and some more stuff found on google, but didn't work. I think the easiest would be removing the tooltips for now, so it will work. We can investigate further later if we thing tooltips are necessary there.